packages/components/Menu/src/MenuItem/MenuItemTokens.macos.ts

```diff
@@ -9,7 +9,7 @@ export const defaultMenuItemTokens: TokenSettings<MenuItemTokens, Theme> = (t: T
   checkmarkSize: 16,
   color: t.colors.menuItemText, // matches ContextualMenu
   fontFamily: t.typography.families.primary,
-  fontSize: globalTokens.font.size300,
+  fontSize: 13, // aligning with ContextualMenu font size
```

Comment:
Is it worth noting that, not only are we aligning to ContextualMenu, but also to the native macOS menu?

Reply:
We have the same comment :D

Reply:
Confirmed with HIG that 13 is correct. Probably why I used it for ContextualMenu. If you wanted to be very exact, you could also check with this: https://developer.apple.com/documentation/appkit/nsfont/1533068-menufont

Reply:
@Saadnajmi Thank you for checking! I'm thinking replacing ContextualMenu with just NSMenu assuming that CM will soon be deprecated?

Reply:
Yeah that sounds fine. Remove references to CM and replace with NSMenu